internal/generator/services.go

```diff
@@ -35,6 +35,26 @@ var supportedAutogeneratedTypes = []string{
 	"python",
 }
 
+// these are lagoon types that come with resources requiring backups
+var typesWithBackups = []string{
```

Comment:
Need some eyes on this to make sure that these are the types that have something that requires backups

Reply:
looks complete to me - cli-persistent and worker-persistent have no need for backups as they are exclusively using other services storage.